src/gantry/screens.py

```
@@ -1110,469 +1111,46 @@ def on_search_input_search_changed(self, message: SearchInput.SearchChanged) ->
-
-        # Existing keybindings
-        ("tab", "app.action_switch_screen", "Switch to Cluster View"),
-        Binding("slash", "focus_search", "Search", priority=True),
-        ("c", "show_context_picker", "Pick Context"),
-        ("enter", "execute_action('deploy')", "Deploy Chart"),
-        ("r", "refresh_charts", "Refresh"),
-        ("q", "quit", "Quit Gantry"),
+        ("tab", "app.action_switch_screen", "Cluster View"),
+        ("r", "action_refresh", "Refresh"),
+        ("q", "quit", "Quit"),
     ]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)